spec/factories/document_type_factory.rb

```diff
@@ -14,6 +14,7 @@
     contents { [] }
     tags { [] }
     guidance { [] }
+    topics { false }
```

Comment:
Shouldn't this also default to `true`?

Reply:
It needs to be false. I mentioned in the commit that we'll need loads of API stubs otherwise, because all the tests will start running with topic API calls enabled.